Ex_22/index1.js

```diff
@@ -0,0 +1,12 @@
+'use striect'
+let curHour = Number(prompt ('Please, enter fwtat hour is it (from 0-23)', 100));
+if (curHour < 0 || curHour > 23) {
+    alert ("Your input incorrect");
+}
+else if (curHour < 8 || curHour == 18 ) {
+    alert ('It is not working hours');
+}
+else (curHour > 8 || curHour == 8 && curHour < 18) ;
```

Comment:
Here you are using too many unnecessary conditions, Natali.
You can do this in much more simple way. Please think about it and fix it.